src/legacy/ui/public/vis/vis_types/_variables.scss

```diff
@@ -0,0 +1,5 @@
+// TODO: Use the same styles for TSVB and Vislib vis'
+// Duplicate of src/legacy/core_plugins/kbn_vislib_vis_types/public/vislib/_variables.scss
```

Comment:
Again, don't duplicate this file, just import the correct path of a single file.